NetCoreRestApi/DataLayer/Repositories/CategoryRepository.cs

```diff
@@ -2,10 +2,20 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace BusinessLayer.Manager
+namespace DataLayer.Repositories
 {
-    public class CategoryManager : ICategoryManager
+    public class CategoryRepository : IRepository<CategoryModel>
     {
+        public void Create(CategoryModel entity)
```

Comment:
Order CRUD methods of IRepository and Repository classes in the following way
GetById
GetAll
Create
Update
Delete

Reply:
done